src/legacy/core_plugins/embeddable_api/README.md

```diff
@@ -0,0 +1,31 @@
+# The Embeddable API V2
```

Comment:
Would be nice to add to README a small code example.

Reply:
I didn't want to do that because then it can quickly get out of date and it's very easy to forget to update it.  By pointing to real code samples that are being tested, we guarantee that won't happen.